components/match2/wikis/ageofempires/legacy/legacy_bracket_match_summary.lua

```diff
@@ -0,0 +1,56 @@
+---
+-- @Liquipedia
+-- wiki=ageofempires
+-- page=Module:LegacyBracketMatchSummary
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Arguments = require('Module:Arguments')
+local Array = require('Module:Array')
+local Json = require('Module:Json')
+local Table = require('Module:Table')
+
+local LegacyBracketMatchSummary = {}
+
+---@param args table
+---@return table
+function LegacyBracketMatchSummary._handleMaps(args)
+	Array.mapIndexes(function(index)
+		local prefix = 'map' .. index
+
+		--Template:MatchTeam
+		if args['match' .. index] then
+			local match = Json.parse(Table.extract(args, 'match' .. index)) --[[@as table]]
+			Table.mergeInto(args,
+					Table.map(match, function(subKey, value)
+						return prefix .. subKey, value
+					end)
+			)
+			args[prefix] = Table.extract(args, prefix .. 'map')
+			args[prefix .. 'mode'] = Table.extract(args, prefix .. 'mapmode')
+		end
+
+		if args[prefix] then
+			local mapInfo = Array.parseCommaSeparatedString(args[prefix], '|')
```

Comment:
what about the display part?
you currently throw it away

Reply:
This module is only used when var `islegacy` is true

Reply:
There is no support for link and display, [MGI](https://github.com/Liquipedia/Lua-Modules/blob/main/components/match2/wikis/ageofempires/match_group_input_custom.lua#L264), tries to find the correct map display based on wikivars set by Template:Maps or Infobox/League.

Reply:
Should be fine, as old tournaments should already have the displayname put in the infobox as well so fetching it should work in most cases.